src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutViewModel.cs

```diff
@@ -46,10 +50,36 @@ public LoadoutViewModel(IWindowManager windowManager, IServiceProvider servicePr
             .ObserveCountChanged()
             .Select(count => count > 0);
 
-        ViewFilesCommand = hasSelection.ToReactiveCommand<Unit>(_ =>
-        {
-            // TODO:
-        });
+        ViewFilesCommand = Adapter.SelectedModels
+            .ObserveCountChanged()
+            .SelectAwait(async (count, cancellationToken) =>
+                {
+                    if (count != 1) return false;
+
+                    var group = await GetViewModFilesLoadoutItemGroup(cancellationToken);
+                    return group.HasValue;
+                },
+                AwaitOperation.Drop
+            )
+            .ToReactiveCommand<NavigationInformation>(async (info, cancellationToken) =>
+                {
+                    var group = await GetViewModFilesLoadoutItemGroup(cancellationToken);
+                    if (!group.HasValue) return;
```

Comment:
I don't like this approach, as we're doing the same thing twice. Instead, have a subscription to the selected items, that calculates whatever we need. The can-execute observable would be a second observable that checks whether we have a calculated value or not.